src/main/scala/faces/utils/Helpers.scala

```diff
@@ -91,23 +91,62 @@ case class Helpers(cfg: FacesSettings)(implicit rnd: Random) {
     new File(bgPath).listFiles.filter(_.getName.endsWith(bgType)).toIndexedSeq
   }
 
-  // center the face around a point between the ears and nose (change here if you want to center another point)
-  def center(rps: RenderParameter): RenderParameter = {
-    val lmLeftEar: TLMSLandmark2D = renderer.renderLandmark("left.eye.pupil.center", rps).get
-    //ear.tragus.tip", rps).get // get is safe, checked during construction
-    val lmRightEar: TLMSLandmark2D = renderer.renderLandmark("right.eye.pupil.center", rps).get
+  // center the face around chin, eyes, nose and ears and crop face box (change here if you want to center another point)
+  def centerFaceBox(rps: RenderParameter, scaling : Double = 1): RenderParameter = {
+    val lmTop: TLMSLandmark2D = renderer.renderLandmark("center.front.trichion", rps).get
```

Comment:
This could be written a bit more readable:

```
val lmTags = Seq(
    "Tag1",
    "Tag2",
    ...)
val lms = lmTags.map(tag => renderer.renderLandmark(tag, rps).get )

val xmax = lms.map(_.point.x).max
....
....
```